src/extensions/default/InlineBorderRadiusEditor/main.js

```diff
@@ -66,11 +67,14 @@ define(function (require, exports, module) {
 
             // Get the css property name after removing spaces and ":" so that we can check for it in the file BorderRadiusProperties.json
             cssPropertyName = cursorLine.split(':')[0].trim();
-
+            
             if (!cssPropertyName || !properties[cssPropertyName]) {
                 return null;
             }
 
+            singleProperty = BorderRadiusUtils.getSingleProperty(cssPropertyName);
+            isSingleProperty = !!singleProperty;
```

Comment:
you don't need globals for this. Just declare them here and return them in the object returned. Then, in the calling function, use `context.cssClassSuffix`, etc. to access these values.